snappy_wrappers/wrappers/pvactools/pvacseq/wrapper.py

```diff
@@ -0,0 +1,159 @@
+import os
+
+from snakemake import shell
+
+__author__ = "Pham Gia Cuong"
+__email__ = "pham.gia-cuong@bih-charite.de"
+
+step = snakemake.config["pipeline_step"]["name"]
+config = snakemake.config["step_config"][step]
+
+algorithms = snakemake.params.args["algorithms"]
+epitope_length = snakemake.params.args["epitope_length"]
+normal_smaple = snakemake.params.args["normal_sample"]
+BINDING_THRESHOLD = f"-b {snakemake.params.args["BINDING_THRESHOLD"]} "
+percentile_threshold = (
+    f"--percentile-threshold {snakemake.params.args["percentile_threshold"]} "
+    if snakemake.params.args["percentile_threshold"] is not None
+    else ""
+)
+
+
+allele_specific_binding_thresholds = (
+    "--allele-specific-binding-thresholds "
+    if snakemake.params.args["allele_specific_binding_thresholds"]
+    else ""
+)
+
+aggregate_inclusion_binding_threshold = (
+    f"--aggregate-inclusion-binding-threshold {snakemake.params.args["aggregate_inclusion_binding_threshold"]} ",
+)
+
+netmhc_stab = "--netmhc-stab " if snakemake.params.args["netmhc_stab"] else ""
+
+NET_CHOP_THRESHOLD = (f"--net-chop-threshold {snakemake.params.args["NET_CHOP_THRESHOLD"]} ",)
+
+PROBLEMATIC_AMINO_ACIDS = (
+    f"--problematic-amino-acids {snakemake.params.args["PROBLEMATIC_AMINO_ACIDS"]} ",
+)
+
+# if {snakemake.params.args["run_reference_proteome_similarity"]}:
+#     run_reference_proteome_similarity= f"--run-reference-proteome-similarity",
+
+FAST_SIZE = (f"-s {snakemake.params.args["FAST_SIZE"]} ",)
+
+exclude_NAs = "--exclude-NAs " if snakemake.params.args["exclude_NAs"] else ""
+
+NORMAL_COV = (f"--normal-cov {snakemake.params.args["NORMAL_COV"]} ",)
+TDNA_COV = (f"--tdna-cov {snakemake.params.args["TDNA_COV"] }",)
+TRNA_COV = (f"--trna-cov {snakemake.params.args["TRNA_COV"]} ",)
+NORMAL_VAF = (f"--normal-vaf {snakemake.params.args["NORMAL_VAF"]} ",)
+
+maximum_transcript_support_level = (
+    f"--maximum-transcript-support-level {snakemake.params.args["maximum_transcript_support_level"]} ",
+)
+
+pass_only = "--pass-only " if snakemake.params.args["pass_only"] else ""
+
+tumor_purity = (
+    f"--tumor-purity {snakemake.params.args["tumor_purity"]} "
+    if snakemake.params.args["tumor_purity"] is not None
+    else ""
+)
+
+op_dir = "/".join(snakemake.output.all_epitopes.split("/")[:-2])
+
+files_to_bind = {
```

Comment:
I am not sure that currently you need to bind the any directory. My understanding is that the `pvacseq` command requires only files within the step directory, both for input and output.